Prox/Prox/Widgets/Toast.swift

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+class Toast {
+    private let text: String
+
+    init(text: String) {
+        self.text = text
+    }
+
+    func show() {
+        guard let window = UIApplication.shared.windows.first else { return }
+
+        let toast = UIView()
+        toast.alpha = 0
+        toast.backgroundColor = Colors.toastBackground
+        toast.layer.cornerRadius = 18
+        window.addSubview(toast)
+
+        let label = UILabel()
+        label.text = text
+        label.textColor = Colors.toastText
+        label.font = Fonts.toast
+        label.numberOfLines = 0
+        toast.addSubview(label)
+
+        toast.snp.makeConstraints { make in
+            make.bottom.equalTo(window).offset(-30)
+            make.centerX.equalTo(window)
+            make.leading.greaterThanOrEqualTo(window)
```

Comment:
Because we also have the `label` constraint: `make.leading.trailing.equalTo(toast).inset(20)`

Combined, these constraints will make the toast grow with the label until it reaches the screen width, at which point they'll force a line break.